src/commands/PublishCommand.js

```diff
@@ -542,6 +542,8 @@ export default class PublishCommand extends Command {
 
     this.updates.forEach((update) => {
       this.execScript(update.package, "prepublish");
+      this.execScript(update.package, "prepare");
```

Comment:
Actually, `this.execScript` is used for executing custom, non-npm lifecycle scripts that are just `.js` files living in a package-local `scripts` directory. They're a lerna-specific extension that allows you to do things _outside_ the normal npm lifecycles. Admittedly, not something I use, but largely inherited from the babel project.

I don't think we should extend this artificial pattern to the extra names, they're just arbitrary and never called during the normal `npm publish` lifecycles. `npm publish` is being called (in topological order) in `NpmUtilities.publishTaggedInDir` and it should automatically handle running the various lifecycles where appropriate, I think.

```
package-foo/
 - package.json
 - scripts/
    - prepublish.js
    - postpublish.js
 - src/
    - index.js
```